ote/src/cljc/ote/transit_changes.cljc

```diff
@@ -138,6 +138,48 @@
 (defn earliest-stop-sequence [stop]
   (:gtfs/stop-sequence stop))
 
+(defn interval->seconds
```

Comment:
This should be put into time.cljc - coincidentally I accidentally did it in https://github.com/finnishtransportagency/mmtis-national-access-point/pull/1159 :)

Reply:
Merge your pr and I'll update my pr to use that fn.